### PR TITLE
Fix the workflow again to push verified tags

### LINKED ISSUE #9: Fix workflow to push verified tags

### PR DESCRIPTION
**Pull request by SagaRut**
Sorry this is hopefully the last edit to these

### CODE REVIEW

File: .github/workflows/release.yml
Comment (Timdnb) [inline .github/workflows/release.yml:1]: why did you remove the - main branch? For the rest it looks good (assuming the GH_TOKEN also works)

Comment (SagaRut) [reply .github/workflows/release.yml:1]: ah forgot to push it back

**Review by Timdnb — APPROVED** [review]
